authpass/lib/ui/screens/password_list.dart

```diff
@@ -923,11 +939,49 @@ class _PasswordListContentState extends State<PasswordListContent>
     return null;
   }
 
+  List<Widget>? _buildAutofillSuggestBanners() {
+    if (_autofillStatus == null ||
+        _autofillStatus != AutofillServiceStatus.disabled) {
+      return null;
+    }
+    if (_dismissedAutofillSuggestion != null &&
+        _dismissedAutofillSuggestion == true) {
```

Comment:
```suggestion
    if (_dismissedAutofillSuggestion == true) {
```
the check for `!= null` feels a bit redundant